deploy/charts/distr/values.yaml

```diff
@@ -239,6 +246,12 @@ postgresql:
     database: distr
     existingSecret: ''
   architecture: standalone
+  primary:
```

Comment:
I'm not sure this is a good idea to keep around. Is there a specific reason why you added this?

Reply:
For clarification, the password is set correctly, and the chart is using that, however what i found when installing it on a cluster is that the db rejected the connections, as they were coming from an untrusted source. The added configurations allows access from all sources. 
This can be tested by creating a local cluster, installing the chart and verifying if the hub can speak to the db. Dunno if you have such a test in the CI, or are interested in adding it :)